pyproject.toml

```diff
@@ -177,6 +177,10 @@ pythonpath = [
   "tests/plugins/data/test-plugin",
 ]
 
+[tool.mypy]
+mypy_path = "conda"
+modules = ["conda"]
```

Comment:
Do we need to also do this for `conda_env`?

Reply:
`mypy` from this PR